src/algokit_utils/__init__.py

```diff
@@ -80,6 +80,7 @@
     OnCompleteCallParametersDict,
     RawTransactionParameters,  # noqa: ignore[F401]
     TransactionParameters,
+    TransactionParametersDict,  # : ignore[F401]
```

Comment:
Should add this to models.py too, to avoid the ruff warning (the above ignores have it as they were for deprecated types)